src/test/regress/sql/multi_tenant_isolation.sql

```diff
@@ -9,6 +9,18 @@ SELECT nextval('pg_catalog.pg_dist_placement_placementid_seq') AS last_placement
 \gset
 ALTER SEQUENCE pg_catalog.pg_dist_placement_placementid_seq RESTART 100000;
 
+CREATE OR REPLACE FUNCTION public.wait_for_resource_cleanup() RETURNS void AS $$
+DECLARE
```

Comment:
I think it's better to put this definition in `multi_test_helpers.sql`. That way we'll be able to continue to run all tests with `check-minimal`.